internal/security/bootstrapper/redis/configure.go

```diff
@@ -63,8 +62,8 @@ func Configure(ctx context.Context,
 		nil,
 		startupTimer,
 		dic,
+		true,
```

Comment:
Isn't there a way to use a named constant here.  "true" and "false" is so non-self-explanatory.

Reply:
that is self-explanatory in go-mod-bootstrap's API IMO.

Reply:
this is calling go-mod-bootstrap's API btw

Reply:
@bnevis-i , there reason I don't think it is an issue is because this is what my IDE display for me:
![image](https://user-images.githubusercontent.com/44779287/113938243-29b91b00-97af-11eb-8586-e75baa9396fd.png)

Reply:
Sweet.